pkg/rtc/participant.go

```diff
@@ -1567,10 +1576,17 @@ func (p *ParticipantImpl) addPendingTrackLocked(req *livekit.AddTrackRequest) *l
 		})
 	}
 
+	p.params.Telemetry.TrackPublishRequested(context.Background(), p.ID(), p.Identity(), ti)
+	p.supervisor.AddPublication(livekit.TrackID(ti.Sid), ti.Type, func(t types.LocalMediaTrack) {
```

Comment:
consolidated logic below, and computing telemetry from monitor onSuccess.

Reply:
Same comment as above, can remove ti.Type